google/cloud/spanner/timestamp.cc

```diff
@@ -25,7 +26,7 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace {
 
 Status OutOfRange(std::string message) {
-  return Status(StatusCode::kOutOfRange, std::move(message));
+  return internal::OutOfRangeError(std::move(message), GCP_ERROR_INFO());
```

Comment:
In this case it seems like we would want the caller to give us the error info.

Reply:
Done